modules/react/src/utils/extract-jsx-layers.js

```diff
@@ -1,27 +1,62 @@
-import React from 'react';
+import React, {createElement} from 'react';
 import {inheritsFrom} from './inherits-from';
-import {Layer} from '@deck.gl/core';
+import {Layer, View} from '@deck.gl/core';
+
+// recursively wrap render callbacks in `View`
+function wrapInView(node) {
```

Comment:
I am missing something, looks to me like all children are being wrapped in views, not just render callbacks.